airbyte-webapp/src/components/StreamTestingPanel/TestControls.tsx

```diff
@@ -0,0 +1,40 @@
+import { FormattedMessage } from "react-intl";
+
+import { RotateIcon } from "components/icons/RotateIcon";
+import { Button } from "components/ui/Button";
+import { Text } from "components/ui/Text";
+
+import { useConnectorBuilderState } from "services/connectorBuilder/ConnectorBuilderStateService";
+
+import { ConfigMenu } from "./ConfigMenu";
+import styles from "./TestControls.module.scss";
+
+interface TestControlsProps {
+  onClickTest: () => void;
+}
+
+export const TestControls: React.FC<TestControlsProps> = ({ onClickTest }) => {
+  const { selectedStream } = useConnectorBuilderState();
+
+  return (
+    <div className={styles.container}>
+      <ConfigMenu />
+      <div className={styles.urlDisplay}>
+        <Text size="lg">{selectedStream.url}</Text>
+      </div>
+      <Button
+        className={styles.testButton}
+        onClick={onClickTest}
+        icon={
+          <div>
```

Comment:
This feels weird that this is needed, and sounds like a bug in our Button component. Maybe you could share that with the platform workflow team that they could have a look if we can fix the Button component to not require this wrapped `div` anymore. (Shouldn't block merging this though!)

Reply:
Created this issue: https://github.com/airbytehq/airbyte/issues/18581